### PR TITLE
ME16: Synced SVD files to register files.

### PR DESCRIPTION
**Pull request by sihyung-maxim**
Changes:
- Updated max32675.svd.
- Updated register files to pass linter checks.
- ECC -> TRIMSIR registers, but sticking with ECC name.
- Updated copyright notices.

### CODE REVIEW

**Comment by sihyung-maxim** [comment]
There are linter errors in the USB examples and other header files unrelated to the changes in this PR... Need to check it out in the morning.

File: Libraries/CMSIS/Device/Maxim/MAX32675/Include/max32675.svd [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <device xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schemaVersion="1.1" xsi:noNamespaceSchemaLocation="svd_schema.xsd">
  <vendor>Maxim-Integrated</vendor>
  <vendorID>Maxim</vendorID>
- <name>MAX32675</name>
+ <name>max32675</name>
  <series>ARMCM4</series>
```
Comment (lorne-maxim) [inline Libraries/CMSIS/Device/Maxim/MAX32675/Include/max32675.svd:6]: Is the lower case part name consistent with our other parts?

**Review by lorne-maxim — APPROVED** [review]

File: Libraries/CMSIS/Device/Maxim/MAX32675/Include/max32675.svd (hunk 1 above)
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32675/Include/max32675.svd:6]: Yes, I've been making this lowercase as I go through all our parts to pass ARM's SVDConv checks.

**Review by Jake-Carter — APPROVED** [review]

**Comment by sihyung-maxim** [comment]
Holding off on merging until I get confirmation on some missing FCR_AUTOCAL registers.

**Comment by sihyung-maxim** [comment]
I'm going to merge in the changes now but create another PR just for the AUTOCAL registers once they are verified and ready to be added.

**Pull request merged**